docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
+description: Learn more about searching and broadcasting available plugins for Porter
```

Comment:
Suggestion: "Find a plugin for Porter and list your own plugin in our search results"

Again just trying to use words that people will use when searching on the web.